test/es6/regex-symbols.js

```diff
@@ -396,8 +425,184 @@ var tests = [
         name: "RegExp.prototype[@@match] has the correct length",
         body: verifySymbolMethodLength.bind(undefined, 1, Symbol.match)
     },
+    {
+        name: "RegExp.prototype[@@match] should run the built-in 'match' when none of the observable properties are overridden",
+        body: function () {
+            var pattern = "(a)-";
+
+            var nonGlobalRe = new RegExp(pattern);
+            var nonGlobalInput = "-a-a-";
+            var result = nonGlobalRe[Symbol.match](nonGlobalInput);
```

Comment:
There is. In `regex-symbols.js`, `verifySymbolMethodName` gets called for `RegExp.prototype[@@match]` to verify the name.